### DISCUSSION TITLE
JFS and MinIO Object Storage w/ Tiering

### DISCUSSION THREAD

**Discussion by ghost** [juicedata/juicefs#3192] [Q&A] [answered]
Has anyone tried or utilized this in practice? Are there any caveats? i.e does fsck pull everything back down?  I have the ability to POC it, but figured I would see if anyone had any experience with this at all before, I go down that road and find out I waisted my time.

**Comment by davies** [comment, marked as answer]
JuiceFS can't use tiered storage right now.

During fsck, it will list all the objects from MinIO and compare it with the results from meta engine to find out the difference.
